dashboard/app/models/course_version.rb

```diff
@@ -89,4 +95,9 @@ def destroy_and_destroy_parent_if_empty
   def contained_lessons
     units.map(&:lessons).flatten
   end
+
+  def all_standards_url
+    prefix = content_root_type === 'UnitGroup' ? 'courses' : 's'
+    "/#{prefix}/#{name}/standards"
```

Comment:
nit: I personally prefer using the Rails routes helpers, in this case `standards_course_path` and `standards_script_path`, to avoid hardcoding urls in the code